compiler/kernel/proc.go

```diff
@@ -590,47 +586,22 @@ func (b *Builder) PushdownOf(trunk *dag.Trunk) (*Filter, error) {
 	return filter, nil
 }
 
-func (b *Builder) LoadConsts(ops []dag.Op) error {
-	scope := b.scope
-	zctx := b.pctx.Zctx
-	for _, p := range ops {
-		switch p := p.(type) {
-		case *dag.Const:
-			val, err := evalAtCompileTime(zctx, scope, p.Expr)
-			if err != nil {
-				return err
-			}
-			if val.IsError() {
-				return fmt.Errorf("cannot resolve const '%s' at compile time", p.Name)
-			}
-			scope.Bind(p.Name, val)
-		case *dag.TypeProc:
-			name := p.Name
-			typ, err := zson.TranslateType(zctx, p.Type)
-			if err != nil {
-				return err
-			}
-			alias, err := zctx.LookupTypeAlias(name, typ)
-			if err != nil {
-				return err
-			}
-			zv := zed.NewTypeValue(alias)
-			scope.Bind(name, zv)
-		default:
-			return fmt.Errorf("kernel.LoadConsts: not a const: '%T'", p)
-		}
-	}
-	return nil
-}
-
-func evalAtCompileTime(zctx *zed.Context, scope *Scope, in dag.Expr) (*zed.Value, error) {
+func EvalAtCompileTime(zctx *zed.Context, in dag.Expr) (val *zed.Value, err error) {
```

Comment:
Maybe call this `EvalNow`?

Reply:
"Now" is always implied for any function call.  This name is meant to clarify that this is only called without a running flowgraph in expression context without this.  Maybe `Compile AndEvalExprWithoutRuntime` but I kind of prefer it how it is.